### PR TITLE
Support multisig txs in BNS decode

### LINKED ISSUE #1215: Support decoding multisig transactions
As of https://github.com/iov-one/iov-core/pull/1214 we have the ability to post send transactions from a multisig contract. But we currently can't correctly decode such transactions.

### PR DESCRIPTION
**Pull request by willclarktech**
Closes #1215

### CODE REVIEW

File: packages/iov-bns/src/decode.spec.ts [hunk 1]
```diff
@@ -394,6 +396,54 @@ describe("Decode", () => {
...
+              ed25519: fromHex("c9df7bcba2238bedcc681e8b17bb21c1625d21d285b70c20cf53fdd473db9dfb"),
+            },
+            signature: {
+              ed25519: fromHex(
+                "7e899078b35b2d301893b82b0d840f4e1ceb715a86f4d28b2e88312ed605fea83d6b37f7ced4c8023a0d41f79ac9d5556df72c5ec58e26d7db4a6f8c6a537d0a",
+              ),
+            },
+          },
+        ],
+        multisig: [fromHex("8877665544332211")],
```
Comment (webmaster128) [inline packages/iov-bns/src/decode.spec.ts]: Question: is 8 bytes the real-workd length to expect?

File: packages/iov-bns/src/decode.ts [hunk 2]
```diff
@@ -471,7 +472,7 @@ function parseSendTransaction(
   return {
     ...base,
     kind: "bcp/send",
-    sender: identityToAddress(base.creator),
+    sender: encodeBnsAddress(prefix, ensure(msg.source, "source")),
```
Comment (webmaster128) [inline packages/iov-bns/src/decode.ts]: Nice bugfix

File: packages/iov-bns/src/decode.spec.ts (hunk 1 above)
Comment (willclarktech) [reply packages/iov-bns/src/decode.spec.ts]: Yes, the IDs are sequential and covered by https://github.com/iov-one/weave/blob/22678889d3e1750c21574c340aebf8c7f09a7679/orm/sequence.go#L62-L78

Comment (webmaster128) [reply packages/iov-bns/src/decode.spec.ts]: Oh, that's interesting. Then we might even want to encode them numerically in https://github.com/iov-one/iov-core/issues/1216 like we do with proposals.

If they are sequential, we can use `number`s in the safe integer range and throw an error for anything else.

**Pull request merged**